pdf/pdfcanvas.cpp

```diff
@@ -113,9 +113,16 @@ PDFCanvas::PDFCanvas(QQuickItem* parent)
 
 PDFCanvas::~PDFCanvas()
 {
-    if(d->resizeTimer)
-        delete d->resizeTimer;
+    for( int i = 0; i < d->pageCount; ++i )
```

Comment:
The code in this repository doesn't have consistent style on many aspects. Loop and brace style I didn't yet touch and this loop is now consistent with similar existing one.